pabi_budget_drilldown_report/xlsx_template/xlsx_template_wizard.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
+
+      # Budget Drilldown
+      <!-- Budget Drilldown Unit Based -->
+      <act_window id="action_budget_drilldown_unit_export_xlsx_template"
```

Comment:
This action is not only for unit base. Please name it "action_budget_drilldown_report_xlsx"